week-2-practise/level-3/2-validation.js

```diff
@@ -6,7 +6,21 @@
 
 var excludedNums = [6, 14, 91, 111];
 
-function validate(num) {}
+function validate(number) {
+  function isNumber() {
+    return typeof number === "number";
+  }
+  function isInteger() {
+    return Number.isInteger(number);
+  }
+  function isNotInclude() {
+    return !excludedNums.includes(number);
+  }
+  return isNumber() && isInteger() && isNotInclude();
+
+  // If we don't want to use small functions we can write it this way
+  // return (typeof number === "number" && Number.isInteger(number) &&!excludedNums.includes(number));
+}
```

Comment:
I'd argue that in this instance, it would be cleaner to not use the functions. Functions are intended to be reusable, but in this instance, they would not be accessible to any code outside the `validate()` function, so not reusable. Also, your code would be more concise.